src/MessagePack.UnityClient/README.md

```diff
@@ -0,0 +1,11 @@
+# UnitySandbox instruction
+
+Dependencies are managed by [NuGetForUnity](https://github.com/GlitchEnzo/NuGetForUnity).
+
+Before launching Unity, you need to run the following command in this directory to restore the dependent libraries:
+
+> dotnet tool restore
+
+> dotnet nugetforunity restore
+
+Also, since the MessagePack library for development is referenced from `bin\MessagePack\Debug\netstandard2.1`, you need to generate the DLL with a debug build beforehand.
```

Comment:
Yes, we need to build the main DLL itself (otherwise we'd have to upload to NuGet every time during development!)
Would it have been clearer if I had written that as a procedure, you need to run `dotnet build -c DEBUG` in the sln folder?
(By the way, the GitHub Actions CI follows exactly this procedure)

For releases, users will:
* Reference the pre-built dll from NuGet for `MessagePack/MessagePackAnalyzer` using NuGetForUnity
* Reference `src/MessagePack.UnityClient/Assets/Scripts/MessagePack/` as source code in UPM

Reply:
You can change the doc if you feel it necessary. Don't mind my ignorant questions too much as I'm just not familiar with unity.
But I thought one of the value props from this new repo structure was that unity users would _always_ consume MessagePack.dll from the nuget package rather than build it themselves. I thought that was why we would be able to use the latest C# lang version going forward. 
Since unity users don't have to rebuild from source for dependencies of messagepack.dll, why is it that they have to build messagepack.dll itself?

Reply:
Or are you talking about running the messagepack tests within unity as part of our own local validation or test pipeline? If that's the case, and if that doesn't really require users to build messagepack.dll themselves, then I'd like to explore how we can build from source for our own testing while providing the necessary .unitypackage for outside this repo to consume the messagepack.dll as a binary and get a few source files necessary for unity environments.

Reply:
Yes, Unity users don't need to build, as they obtain the built dll from the official NuGet.

However, the developers of this repository are different; the dll under development doesn't exist on NuGet.
I think it's natural to use what's been built locally.
In regular .NET, a separate csproj (MessagePack) associated with the csproj would
automatically build and generate a DEBUG build dll to be retrieved.
This is what we want to do in this Unity project, and since
there's no means to automatically resolve dependencies and build, we're building manually.
Although it's manual, during normal development, you'd typically run the .NET side before looking at the Unity side,
so Unity would almost automatically get the latest build of the dll under development.

By the way, .unitypackage is completely irrelevant.
In this new structure, no .unitypackage should appear at all.